moabb/datasets/ssvep_nakanishi.py

```diff
@@ -40,13 +40,25 @@ def __init__(self):
         super().__init__(
             subjects=list(range(1, 10)),
             sessions_per_subject=1,
-            events={'9.25': 1, '11.25': 2, '13.25': 3, '9.75': 4, '11.75': 5,
-                    '13.75': 6, '10.25': 7, '12.25': 8, '14.25': 9,
-                    '10.75': 10, '12.75': 11, '14.75': 12},
+            events={
+                '9.25': 1,
+                '11.25': 2,
+                '13.25': 3,
+                '9.75': 4,
+                '11.75': 5,
+                '13.75': 6,
+                '10.25': 7,
+                '12.25': 8,
+                '14.25': 9,
+                '10.75': 10,
+                '12.75': 11,
+                '14.75': 12,
+            },
```

Comment:
Skipped adding `fmt: off/on` to this one, as it wasn't *that* bad (and meaningfully easier to read)

Reply:
agreed also